test/rest.test.js

```diff
@@ -1571,6 +1571,41 @@ describe('strong-remoting-rest', function() {
           .end(expectErrorResponseContaining({ message: 'an error' }, done));
       });
 
+      it('should return 500 if an array of errors is thrown', function(done) {
+        var duplicateError = new Error('duplicate error');
+        var errArray = [duplicateError, duplicateError];
+
+        var expectedMessage = 'Failed with multiple errors, see `details` for more information.';
+
+        function method(error) {
+          return givenSharedStaticMethod(function(cb) {
+            cb(error);
+          });
+        }
+
+        request(app).get(method(duplicateError).url)
```

Comment:
I find this confusing. In the context of this request, `duplicateError` is not really duplicated. I am proposing to change it to

```js
var anError = new Error('expected test error');
// or
var testError = new Error('expected test error');
```